extensions/cornerstone/src/panels/PanelMeasurement.tsx

```diff
@@ -48,13 +54,34 @@ export default function PanelMeasurementTable({
 
   const additionalFilter = filterAdditionalFinding(measurementService);
 
-  const { measurementFilter: trackedFilter } = measurementFilters;
-  const measurements = displayMeasurements.filter(
-    item => !additionalFilter(item) && trackedFilter(item)
-  );
-  const additionalFindings = displayMeasurements.filter(
-    item => additionalFilter(item) && trackedFilter(item)
-  );
+  const { measurementFilter } = measurementFilters;
+
+  const defaultGroupingFunction = groupByStudy(displaySetService);
```

Comment:
I think you should assume that any other grouping function might also need services and/or commands managers.  If you do it this way, then you can setup the grouping function appropriately:
const groupingFunction = (groupingFunctionConstructor || groupByStudyConstructor)(props)
where props is the props to the panel measurement table.